recipes/anaquin/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# download and copy required dependencies
+# klib
+wget https://github.com/attractivechaos/klib/archive/spawn-final.tar.gz
```

Comment:
Anaquin only requires random generator from klib, and it's a header file. I can directly embed the file into Anaquin (so no external dependency) if that's the easiest solution.